hardhat.network.ts

```diff
@@ -29,15 +24,25 @@ const networks: HardhatUserConfig['networks'] = {
       mnemonic,
     },
   },
-  avalancheMainnet: {
+  polygon: {
+    chainId: 137,
+    url: process.env.POLYGON_RPC_URL,
+    accounts: {
+      mnemonic,
+    },
+  },
+};
+
+if (!!avalanche) {
```

Comment:
why is this conditional?  this was just fixed

Reply:
It's a conditional so Hardhat doesn't error if the `.envrc` is not setup with a provider URL.